server/handlers/handlers.go

```diff
@@ -82,7 +81,7 @@ func New(plugin PluginIface) *ActivityHandler {
 }
 
 func (ah *ActivityHandler) Start() {
-	ah.quitting.Store(false)
+	ah.quit = make(chan bool)
```

Comment:
Previously discussed with @agnivade to move away from a queue that has to be sent a message per worker to just a `close(ah.quit)` and recreating on `Start`.